### DISCUSSION TITLE
新版本更新地址无法进入

### DISCUSSION THREAD

**Discussion by Maxwell1312** [ZuodaoTech/everyone-can-use-english#986] [Q&A]
新版本的更新，点击下载后，网址一直报错，进不去
![捕获](https://github.com/user-attachments/assets/4d2ff5be-e6b8-445b-95fe-5471fa04a174)

**Comment by an-lee** [comment]
想办法科学上网一下试试？

**Reply by Maxwell1312** [reply]
已解决，只需把网络切换到手机热点就Ok
